Makefile

```diff
@@ -17,9 +17,15 @@ ci:
 	@echo "Build project"
 	@npm run build
 
-deploy: configure-docker deploy-image run-migrations
-	@kubectl -n gitlint-bot apply -f ./deploy/k8s-deployment.yaml
-	@kubectl -n gitlint-bot set image deployment/gitlint-bot-production gitlint-bot-production=${APP_IMAGE_SHA}
+deploy: decrypt-blackbox configure-docker deploy-image
```

Comment:
Can you refactor this into two targets?
- `deploy-chart`: run helm to deploy the chart
- `deploy`: run the four targets sequentially

This allow performing each steps independently at local environment.